bin/oneoff/import_unit_details.rb

```diff
@@ -83,6 +83,8 @@ def main(options)
     script = Script.find_by_name!(unit_name)
     log "found code studio script name #{script.name} with id #{script.id}"
 
+    raise "Trying to import a script that is marked as stable" if script.is_stable
```

Comment:
can we change this to check that the script is not `hidden` instead, here and below? all stable scripts should be visible, but checking for visible will protect scripts like dance party too.

Reply:
Done!